app/assets/stylesheets/application.scss

```diff
@@ -15,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap-sprockets";
+@import "bootstrap";
```

Comment:
@naimaseales as discussed briefly on call today, consider taking full control of the stylesheet load order with this change:

```diff
- *
- *= require_tree .
- *= require_self
 */
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "welcome";
+@import "users";
```